AppleStore/AppleStore/PageViewController/WomanHealthPageViewController.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  WomanHealthPageViewController.swift
+//  AppleStore
+//
+//  Created by  Евгений on 16.07.2021.
+//
+
+import UIKit
+
+class WomanHealthPageViewController: UIPageViewController {
+    // MARK: - Private Properties
+    private var womanHealthPages = [WomanHealthHelper]()
+    private var numberOfPage = 0
```

Comment:
```suggestion
    private var numberPage = 0
```
оп гайду Of не юзать